config/routes.rb

```diff
@@ -27,5 +32,8 @@
   post 'tenants', to: 'tenants#create'
 
   resources :tenants, as: :tenant, path: '', param: :slug, except: [:destroy]
+  get '/404', to: 'errors#not_found'
+  get '/500', to: 'errors#internal_server_error'
 
+  get '*path', to: 'errors#not_found'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.